samples/subsys/usb/cdc_acm_composite/sample.yaml

```diff
@@ -3,6 +3,7 @@ sample:
 tests:
   sample.usb.cdc-acm-composite:
     depends_on: usb_device gpio
+    platform_exclude: frdm_kl25z
```

Comment:
@finikorg can you try to rebuild this or add a `.conf` file?

Reply:
I added
```
--- a/samples/subsys/usb/cdc_acm_composite/prj.conf
+++ b/samples/subsys/usb/cdc_acm_composite/prj.conf
@@ -11,6 +11,7 @@ CONFIG_USB_DEVICE_PRODUCT="Zephyr CDC ACM sample"
 CONFIG_USB_COMPOSITE_DEVICE=y
 CONFIG_USB_CDC_ACM=y
 CONFIG_USB_CDC_ACM_DEVICE_COUNT=2
+CONFIG_USB_CDC_ACM_RINGBUF_SIZE=512
 
```